README.md

```diff
@@ -18,14 +18,14 @@ This repository houses plugins that Rollup considers critical to every day use o
 |                                       |                                                                                           |
 | ------------------------------------- | ----------------------------------------------------------------------------------------- |
 | [alias](packages/alias)               | Define and resolve aliases for bundle dependencies                                        |
-| [auto-install](packages/auto-install) | Automatically install dependencies that are imported by a bundle                         |
+| [auto-install](packages/auto-install) | Automatically install dependencies that are imported by a bundle                          |
```

Comment:
please revert the formatting in this file

Reply:
sure thing!

Reply:
seems this format job is being run by husky